game/templates/game/board.html

```
@@ -386,9 +386,9 @@
     <div id="rulebookModal"
         style="display:none; position:fixed; inset:0; background:rgba(0,0,0,0.85); z-index:9999; align-items:center; justify-content:center;">
         <div
-            style="position:relative; width:95vw; height:90vh; background:#13161e; border-radius:12px; overflow:hidden;">
+            style="position:relative; width:min(95vw, 1400px); height:90vh; background:#13161e; border-radius:12px; overflow:hidden; padding-top:48px;">
             <button onclick="document.getElementById('rulebookModal').style.display='none'"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)